src/pages/ModelViewPage.tsx

```diff
@@ -137,23 +141,62 @@ const ModelViewPage = () => {
         })
         return
       }
+      if (parsedStress.values.length !== stressValues.length) {
+        openModal({
+          title: t('validation.error'),
+          message: t('validation.stressIsNotTheSameAsNodesCount', {
+            stressCount: parsedStress.length,
+            elementsCount: stressValues.length
+          }),
+          confirmation: t('validation.checkDataAndTryAgain'),
+          buttons: 'ok'
+        })
+
+        return
+      }
 
-      if (otherCharacteristic.values.length !== indicesMatrix.length) {
+      const calculatedMises = calculateMisesStress(parsedStress)
+
+      const stress = buildMisesPhysicalQuantity(calculatedMises)
+
+      dispatch(setStress({ stress, fileName: file.name }))
+      dispatch(setDisplay('stress'))
+    },
+    [dispatch, t, openModal, stress]
```

Comment:
replace `stress` with `stressValues`
```suggestion
    [dispatch, t, openModal, stressValues]
```